AssociativeArraysExercise/01-word-tracker.js

```diff
@@ -0,0 +1,9 @@
+function solve([searchingWordsStr, ...words]) {
+    const searchingWords = searchingWordsStr.split(' ');
+
+    const repetitionsOfWords = words.reduce((obj, word) => ({...obj, [word]: (obj[word] || 0) + 1}), {});
+
+    const sortedSearchingWords = searchingWords.sort((a, b) => (repetitionsOfWords[b] || 0) - (repetitionsOfWords[a] || 0));
```

Comment:
No need for sortedSearchingWords variable. It just make confusion that .sort does not change the array.